docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       POSTGRES_PASSWORD: postgres
 
   axonserver:
-    image: axoniq/axonserver:4.6.10-dev
+    image: axoniq/axonserver:4.6.11-dev
```

Comment:
Can you check what the latest is? The image tags have changed to something like `2023.2.1-jdk-17-nonroot`

Reply:
You are right!
`4.6.11` was the latest version for SE only image.  But they have combined both EE and SE into one image after that release.  Upgraded ours to the latest one now.

Reply:
Removed ✅